student-test/assignment1/Test_validate_username.c

```diff
@@ -14,9 +14,7 @@
 */
 void test_validate_my_username()
 {
-    /**
-     * TODO: Replace the line below with your code here as described above to verify your /conf/username.txt 
-     * config file and my_username() functions are setup properly
-     */
-    TEST_ASSERT_TRUE_MESSAGE(false,"AESD students, please fix me!");
+    const char *username_auto=my_username();
+    char *username_malloc=malloc_username_from_conf_file();
```

Comment:
@shashankchandrasekaran what happens to the string `username_malloc` after this function exits?